pkg/proc/native/ptrace_linux_64bit.go

```diff
@@ -1,4 +1,4 @@
-//go:build (linux && amd64) || (linux && arm64) || (linux && ppc64le)
+//go:build (linux && amd64) || (linux && arm64) || (linux && ppc64le) || (linux && loong64)
```

Comment:
`go:build (linux && (386 || ppc64le || loong64)) || (darwin && arm64) || (windows && arm64)`

I think it's better this way